src/bp-messages/bp-messages-filters.php

```diff
@@ -727,6 +730,28 @@ function bp_recipients_recipient_get_where_conditions_callback( $where_condition
 	return $where_conditions;
 }
 
+/**
+ * Function will check content empty or not for the media, document and gif.
+ * If content will empty then return true and allow empty content in DB for the media, document and gif.
+ *
+ * @since BuddyBoss [BBVERSION]
+ *
+ * @param bool         $validated_content Boolean from filter.
+ * @param string       $content           Message content.
+ * @param array|object $post              Request object.
+ *
+ * @return bool
+ */
+function bb_check_is_message_content_empty( $validated_content, $content, $post ) {
```

Comment:
@milanhirapra This is still wrong. As I requested before we should check for every component into the specific component. What if a particular component is disabled.

Reply:
@dipskakadiya I had added filters, but you suggested to remove those filters.
Please check this commit: https://github.com/buddyboss/buddyboss-platform/pull/2886/commits/27c7cff4d4865b183e4ca8e3fea48468370299d7